### PR TITLE
rename to 'Default encryption module'

### PR DESCRIPTION
**Pull request by jancborchardt**
@schiesbn I guess this doesn’t change the app name, but I didn’t find where that is defined. Please tell me or do it yourself. ;)

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
:-1: see https://github.com/owncloud/core/issues/16080#issuecomment-99229283

**Comment by MorrisJobke** [comment]
> @schiesbn I guess this doesn’t change the app name, but I didn’t find where that is defined.

In the app description file? ;) https://github.com/owncloud/core/blob/master/apps/encryption/appinfo/info.xml#L19

**Comment by jancborchardt** [comment]
@DeepDiver1975 I know it’s not perfect but it’s at least better than using »ownCloud« in the name.

@schiesbn @MorrisJobke done.

File: settings/templates/admin.php [hunk 1]
```diff
@@ -362,7 +362,7 @@
 			<?php
 			if ($_['encryptionReady'] === false && $_['externalBackendsEnabled'] === true) {
 				p('You need to migrate your encryption keys from the old encryption (ownCloud <= 8.0) to the new one. '
-					. 'Please enable the "ownCloud Default Encryption Module" and run \'occ encryption:migrate\'');
+					. 'Please enable the "Default encryption module" and run \'occ encryption:migrate\'');
```
Comment (nickvergessen) [inline settings/templates/admin.php]: needs rebase

**Comment by nickvergessen** [comment]
> @DeepDiver1975 I know it’s not perfect but it’s at least better than using »ownCloud« in the name.

Well that's the group which created this module?

**Comment by schiesbn** [comment]
@nickvergessen this was also my argument, but  @jancborchardt 's argument is also valid https://github.com/owncloud/core/issues/16080#issuecomment-99201161

Anyway, I'm fine with it :+1: (@DeepDiver1975 's concerns regarding the "default" are also valid for the current name, so nothing changes here)

**Comment by jancborchardt** [comment]
@schiesbn can you rebase this? Am afraid there’s conflicting code changes I have no idea about?

@nickvergessen we shouldn’t drop our brand name too often.

**Comment by schiesbn** [comment]
rebased

**Comment by MorrisJobke** [comment]
I'm fine with this too. @DeepDiver1975 What is your opinion now?

**Comment by MorrisJobke** [comment]
:+1: from my side.

@DeepDiver1975 @schiesbn Please review

**Comment by DeepDiver1975** [comment]
Okay - let get this going :+1:

**Pull request merged**